data/strings/strings.txt

```diff
@@ -20551,7 +20553,7 @@
 
   [power_managment_description]
     tags = android,ios
-    en = When automatic mode is selected the application starts to disable battery draining features depending on the current battery charge level
+    en = When automatic mode is selected, battery draining features will be disabled depending on the current battery level.
```

Comment:
Battery draining features  is hard to understand.

```
When automatic mode is selected, the app attempts to use less power when the battery level is below a threshold.
```

But really, change the text to

```
Try to reduce power usage at the expense of some functionality
```
and then have the three choices be
```
Never
When battery is below 50%
Always
```

Reply:
Or just skip the description and show "never/when battery is low/always". 
The `night mode` screen gets this right.

Reply:
agreed and updated :) I won't specify a percentage because idk what 'low battery' is defined as in this case, but can be fixed in future